src/pages/components/FavoriteGrid.qml

```diff
@@ -61,16 +61,8 @@ SilicaGridView {
     delegate: ListItem {
         id: container
 
-        property Item remorse
-
         function remove() {
```

Comment:
I'd also remove this function and move ```bookmarkModel.removeBookmark(url)``` to the FavoriteContextMenu. Matter of taste as currently FavoriteContextMenu is not using ids from elsewhere and that's also good. Not big deal.

For safety, add url parameter for the remove function is you don't remove this. Now url refers to the model.url but it looks quite odd (maybe I have added at some point of time) as call is coming through FavoriteContextMenu. Url is passed to the context menu in the show.